tests/functional/workloads/cnv/test_vm_lifecycle_and_io.py

```
@@ -44,8 +44,14 @@ def test_vm_lifecycle_and_io(self, setup_cnv, project_factory, multi_cnv_workloa
         ) = multi_cnv_workload(namespace=proj_obj.namespace)
         all_vm_list = self.vm_objs_def + self.vm_objs_aggr
+        logger.info(f"Created {len(all_vm_list)} VMs successfully")
 
+        logger.test_step("Restart VMs and verify FIO status")
         for vm_obj in all_vm_list:
+            logger.info(f"Restarting VM '{vm_obj.name}'")
             vm_obj.restart()
             if check_fio_status(vm_obj):
-                log.info("FIO started after restarting VM")
+                logger.info(
+                    f"FIO running successfully on VM '{vm_obj.name}' after restart"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)